bigbluebutton-html5/private/config/settings.yml

```diff
@@ -138,7 +138,7 @@ public:
       captureSharedNotesByDefault: false
       sendInvitationToAssignedModeratorsByDefault: false
       breakoutRoomLimit: 16
-      allowUploadNewDocsInBreakouts: false
+      allowDocsManagementInBreakouts: false
```

Comment:
I am being asked for us to keep the existing behavior (even as we are adding this property. Contrary to my initial request, let's default to true to match BBB 2.7.9 behavior.